### DISCUSSION TITLE
2023/03/29/GORM%E7%9A%84GEN%E6%A8%A1%E5%BC%8F%E5%88%9D%E4%B8%8A%E6%89%8B/

### DISCUSSION THREAD

**Discussion by giscus** [nickxudotme/nickxudotme.github.io#23] [General]
# 2023/03/29/GORM%E7%9A%84GEN%E6%A8%A1%E5%BC%8F%E5%88%9D%E4%B8%8A%E6%89%8B/

最近准备上手一个新项目，然后在选择底层的 ORM 框架 我之前一直用的 GORM ，其实感觉还挺好用的，但是毕竟用多了，这次也想了解下其他的解决方案 之前在和群友讨论 GORM 的时候，我听见有人对它有一些负面的看法，比如说认为 GORM 不算真正的 ORM，因为它很多时候只是在帮你手动拼凑 SQL

https://nickxu.me/2023/03/29/GORM%E7%9A%84GEN%E6%A8%A1%E5%BC%8F%E5%88%9D%E4%B8%8A%E6%89%8B/?highlight=gorm+gen

**Comment by YXZ252426** [comment]
大佬，这个BILIBILI的视频链接失效了啊

**Reply by nickxudotme** [reply]
正确的，文章太久引用很可能就失效了
